pkg/mover/doc_load_test.go

```diff
@@ -0,0 +1,50 @@
+// Copyright 2021 VMware, Inc.
+// SPDX-License-Identifier: BSD-2-Clause
+
+package mover
+
+import (
+	"fmt"
+)
+
+// Load a chart and its dependencies from a local intermediate bundle tarball
+// into a registry and or repository
+func Example_load() {
+	// i.e ./mariadb-7.5.relocated.tgz
+	destinationPath := "%s-%s.relocated.tgz"
+
+	// Initialize the Mover action
+	chartMover, err := NewChartMover(
+		&ChartMoveRequest{
+			Source: Source{
+				Chart: ChartSpec{
+					// The source intermediate bundle path to place the charts and all its dependencies
+					IntermediateBundle: &IntermediateBundle{Path: "helm_chart.intermediate-bundle.tar"},
+				},
+				// no path to hints file as it is already coming inside the intermediate bundle
+				ImageHintsFile: "./image-hints.yaml",
```

Comment:
the hints file is not needed right?

Reply:
yep, I commented and then forgot to remove...